requirements.txt

```diff
@@ -1,3 +1,4 @@
 cairocffi >= 0.9.0
 cffi >= 1.1.0
+dbus-next
```

Comment:
`dbus-next` is an optional requirement so shouldn't be in the main `requirements.txt` file. It can be in `requirements-dev.txt` though.